Dockerfile.example

```diff
@@ -1,23 +1,27 @@
 FROM python:3.11-alpine
 
 ARG ARTIFACT_VERSION
-
 WORKDIR /app
+
+# Copy in requirements files
+COPY ./requirements.txt ./requirements.txt
 COPY ./ansible-galaxy-requirements.yaml ./ansible-galaxy-requirements.yaml
 
-RUN apk add --update --no-cache gcc libc-dev libffi-dev curl vim bash openssh
+# Install required system packages
+RUN apk add --update --no-cache gcc libc-dev libffi-dev openssh
 
-RUN pip install \
-        --pre \
-        --trusted-host 150.254.211.2 \
-        --extra-index-url https://150.254.211.2/artifactory/api/pypi/geant-swd-pypi/simple \
-        goat-lso==${ARTIFACT_VERSION}
+# Install the LSO python package, and additional requirements
+RUN pip install lso==${ARTIFACT_VERSION}
```

Comment:
The example will show the latest version of the `lso` package as an example. Marking this as resolved. Updating this will be done automatically using `bumpversion`